core/src/Cabana_VerletList.hpp

```diff
@@ -375,6 +382,108 @@ struct VerletListBuilder
             } );
     }
 
+    // Neighbor count team (+serial) operator (only used for CSR lists).
+    KOKKOS_INLINE_FUNCTION
+    void
+    operator()( const CountNeighborsTag<TeamOpTag> &,
+                const typename CountNeighborsPolicy::member_type &team ) const
+    {
+        // The league rank of the team is the cardinal cell index we are
+        // working on.
+        int cell = team.league_rank();
+
+        // Get the stencil for this cell.
+        int imin, imax, jmin, jmax, kmin, kmax;
+        cell_stencil.getCells( cell, imin, imax, jmin, jmax, kmin, kmax );
+
+        // Operate on the particles in the bin.
+        std::size_t b_offset = bin_data_1d.binOffset( cell );
+        Kokkos::parallel_for(
```

Comment:
So 1 extra level of parallelism works with HIP but the second does not?

Reply:
Hierarchical reductions don't work with HIP yet